<?xml version="1.0" encoding="utf-8"?>
<xliff version="1.0">
  <file source-language="en" datatype="plaintext" original="EXT:datamints_error_report/Resources/Private/Language/locallang_db" date="2022-02-0314:53:29Z" product-name="datamints_error_report">
    <header>
      <!--This translation file was generated automatically by the "datamints_locallang_builder" extension. Please note that changes to this file can be lost the next time someone triggers an export in the TYPO3 Backend-module-->
    </header>
    <body>
      <trans-unit id="mail.log-entries.title" resname="mail.log-entries.title" approved="yes">
        <!--part of mail template-->
        <source>The following exceptions have been registered since the last email:</source>
      </trans-unit>
    </body>
  </file>
</xliff>
